lib/v2/caareviews/utils.js

```diff
@@ -0,0 +1,48 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const { parseDate } = require('@/utils/parse-date');
+const { art } = require('@/utils/render');
+const path = require('path');
+
+const rootUrl = 'http://www.caareviews.org';
+
+const getList = async (url) => {
+    const response = await got(url);
+    const $ = cheerio.load(response.data);
+    const list = $('#infinite-content > div')
+        .map((_index, item) => ({
+            title: $(item).find('div.title').text().trim(),
+            link: new URL($(item).find('div.title > em > a').attr('href'), rootUrl).href,
+            author: $(item).find('div.contributors').text().trim(),
+        }))
+        .get();
+
+    return list;
+};
+
+const getItems = async (ctx, list) =>
+    await Promise.all(
+        list.map((item) =>
+            ctx.cache.tryGet(item.link, async () => {
+                const detailResponse = await got(item.link);
+                const $ = cheerio.load(detailResponse.data);
+
+                const coverUrl = new URL($('div.cover > a').attr('href'), rootUrl).href;
+                const content = $('div.content.full-review').html();
+                item.description = art(path.join(__dirname, 'templates/utils.art'), {
+                    coverUrl,
+                    content,
+                });
+                $('div.review_heading').remove();
+                item.pubDate = parseDate($('div.header-text > div.clearfix').text());
+
```

Comment:
Please also set `item.doi`

Reply:
This line aims to remove other content to get only time.

Reply:
I mean this
https://github.com/DIYgod/RSSHub/blob/8a567d5cd4a7a10bb602dff4af55fb066e96d06a/lib/v2/elsevier/latest.js#L74-L75

Reply:
Get it. 😂